bluephos/modules/extract.py

```diff
@@ -0,0 +1,143 @@
+import re
```

Comment:
maybe this module could be named `dft_extract` or `orca_extract` or something - this looks pretty specific to the data produced for that